v3api/transcripts.go

```diff
@@ -167,3 +178,10 @@ func (c *Client) GetEditingLink(mediaFileID string, hoursUntilExpiration int) (s
 	}
 	return response.Data, nil
 }
+
+func (c *Client) setAPIKey(apiKeyParam string) string {
```

Comment:
Should we rename this method? I'd expect ``c.set<Thing>`` to mutate ``c.<Thing>``, but that's not what's happening here.

(I'm having a hard time trying to think of a better name though, sorry :speak_no_evil:)

Reply:
@fsouza `determineApiKey`? I definitely see what you mean but it was the best I could do 😬